images/linux/scripts/base/snap.sh

```diff
@@ -5,4 +5,10 @@
 # when they are rolling a new major update out.
 # Hold is calculated as today's date + 60 days
 
+# snapd is started automatically, but during image generation
+# a unix socket may die, restart snapd.service (and therefore dnapd.socket)
+# to make sure the socket is alive.
+
+systemctl restart snapd
+systemctl is-active --quiet snapd
```

Comment:
Why should we check if a service active?

Reply:
@al-cheb I was actually to put a socket service restart there first (now done), so we indeed have no reason to check activity now.